src/main/scala/com/microsoft/hyperspace/actions/RefreshAction.scala

```diff
@@ -60,8 +60,7 @@ class RefreshAction(
     IndexConfig(previousIndexLogEntry.name, ddColumns.indexed, ddColumns.included)
   }
 
-  final override lazy val logEntry: LogEntry =
-    getIndexLogEntry(spark, df, indexConfig, indexDataPath)
+  final override def logEntry: LogEntry = getIndexLogEntry(spark, df, indexConfig, indexDataPath)
```

Comment:
Shall we add a test? Any regression/bug fix should come with a test.

Reply:
thanks, @imback82 , added.